src/main/java/org/team401/lib/Lidar.java

```diff
@@ -33,8 +33,8 @@
 		private Hardware hardware;
 		private Unit unit;
 		private AtomicReference<Data> latestData;
-		private byte[] buffer = {0,0};
-		private byte[] writeBuffer = new byte[1];
+		private byte[] buffer = {0,0},
+			writeBuffer = new byte[1];
```

Comment:
This shouldn't be a problem, but may affect consistency around other parts of the file.  Also, I may be confusing this for C++, but will this actually give us the desired separation, or will this create a bound reference?